module/combat.js

```diff
@@ -6,19 +6,19 @@
  * Apply the dexterity score as a decimal tiebreaker if requested
  * See Combat._getInitiativeFormula for more detail.
  */
- export const _getInitiativeFormula = function() {
+export const _getInitiativeFormula = function () {
     const actor = this.actor;
-    if ( !actor ) return "1d20";
-    const init = actor.data.data.initiative;
+    if (!actor) return "1d20";
+    const moxie = game.settings.get("sgrpg", "useMoxieCombat"); // Use the raw setting here, since it's where the magic happens
```

Comment:
Iam not sure iam fan of this being such global settings. In my experience its more like per scene and even in one scene we already run into need of having both moxie combat being followed by normal one because of some big fail. I would say that ideally at the start of combat it should ask you which type of combat you want to run or something along those lines. At a same time, iam not sure this can be achieved in Foundry. Any ideas?